dotnet/csharp-grpc-load/load/requests/List.cs

```diff
@@ -0,0 +1,94 @@
+using System.Collections.Concurrent;
+using System.Diagnostics;
+using Momento.Protos.CacheClient;
+
+namespace momento_csharp_load_generator.load.requests
+{
+    public static class List
+    {
+        private static string filler = "fillerfillerfillerfillerfillerfillerfillerfillerfillerfillerfillerfillerfil"; // 75
+        private static ConcurrentDictionary<uint, Google.Protobuf.ByteString> elements = new();
+        private static ConcurrentDictionary<uint, Google.Protobuf.ByteString> listNames = new();
+
+        public static Google.Protobuf.ByteString Element(uint i)
+        {
+            return elements.GetOrAdd(i, j => Google.Protobuf.ByteString.CopyFromUtf8($"value {j,8} {filler}")); // + 15 bytes = 25 + 75 bytes = 100 bytes
+        }
+
+        public static Google.Protobuf.ByteString ListName(uint i)
+        {
+            return listNames.GetOrAdd(i, j => Google.Protobuf.ByteString.CopyFromUtf8($"A list {j}"));
+        }
+        public static async Task Add(uint listNumber, uint elementStart, Scs.ScsClient client, RequestUtil util, Stats stats, CancellationToken cancellationToken)
```

Comment:
Let's separate this into `Push` and `Pop` as pop operation is not really an add operation.